docs/controls/WebView.md

```diff
@@ -43,7 +43,7 @@ When compared to the UWP **WebView** control, the **WebView** controls for WPF a
 
```

Comment:
@mcleanbyron @normesta  In prerequisites we define the minimum OS and SDK version; however, @nmetulev pointed out that there is a standard form of this in the doc template. I've included it on L312 for your review